src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,63 @@
 import React from 'react';
 
 import './MovieCard.scss';
+import { Movie } from '../../types/Movie';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
+type Props = {
+  movie: Movie;
+};
+
+export const MovieCard: React.FC<Props> = ({
+  movie: {
+    title,
+    description,
+    imgUrl,
+    imdbUrl,
+  },
+}) => (
+
+  <>
+    <div className="movies">
```

Comment:
The `.movies` block should be in the `MoviesList` component, not in the `MovieCard` component. Remove the div with the className `movies` and adjust the indentation accordingly.